packages/components/nodes/Etherscan/Etherscan.ts

```diff
@@ -0,0 +1,217 @@
+import {
+    ICommonObject,
+    INode, 
+    INodeData, 
+    INodeExecutionData, 
+    INodeParams, 
+    NodeType,
+} from '../../src/Interface';
+import {
+    handleErrorMessage,
+    returnNodeExecutionData,
+    serializeQueryParams
+} from '../../src/utils';
+import axios, { AxiosRequestConfig, Method } from 'axios';
+class Etherscan implements INode {
+    label: string;
+    name: string;
+    type: NodeType;
+    description: string;
+    version: number;
+    icon: string;
+    incoming: number;
+    outgoing: number;
+    actions: INodeParams[];
+    credentials?: INodeParams[];
+    networks?: INodeParams[];
+    inputParameters?: INodeParams[];
+ 
+    constructor() {
+        this.label = 'Etherscan';
+        this.name = 'etherscan';
+        this.icon = 'etherscan.svg';
+        this.type = 'action';
+        this.version = 1.0;
+        this.description = 'Perform Etherscan operations';
+        this.incoming = 1;
+        this.outgoing = 1;
+        this.actions = [
+            {
+                label: 'API',
+                name: 'api',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Get Ether Balance for a Single Address',
+                        name: 'getEtherBalance',
+                        description: 'Returns the Ether balance of a given address.'
+                    },
+                    {
+                        label: 'Get Ether Balance for Multiple Addresses(separated by a comma)',
+                        name: 'getEtherBalanceMulti',
+                        description: 'Returns the Ether balance of the addresses(each address separated by a comma) entered.'
+                    },
+                    {
+                        label: 'Get Contract ABI for Verified Contract Source Codes',
+                        name: 'getContractABI',
+                        description: 'Returns the Contract Application Binary Interface ( ABI ) of a verified smart contract.'
+                    },
+                ],
+                
+                default: 'getEtherBalance'
+            },
+        ] as INodeParams[];
+        this.networks = [
+            {
+                label: 'Network',
+                name: 'network',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Ethereum Mainnet',
```

Comment:
You can use `NETWORK_LABEL.MAINNET` for label and `NETWORK.MAINNET` for value, we have an enum type for that